src/frontend/magnify/src/views/rooms/settings/RoomSettingsView.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import React from 'react';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import { RoomSettingsView } from './index';
+
+export default {
+  title: 'Views/Rooms/RoomSettingsView',
+  component: RoomSettingsView,
+} as ComponentMeta<typeof RoomSettingsView>;
+
+const Template: ComponentStory<typeof RoomSettingsView> = (args) => {
+  return (
+    <RouterProvider
```

Comment:
ditto

Reply:
Yes for the moment the package is too linked to the router. Gotta undo it